CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Remove `NumProcessedLines` metric from FilterStats [#173](https://github.com/AdRoll/baker/pull/173)
 - Add the Crypt filter [#177](https://github.com/AdRoll/baker/pull/177)
 - Bump vmware-go-kcl and gozstd [#181](https://github.com/AdRoll/baker/pull/181)
+- Improve performance of the logline `ToText` and `Parse` method for low number of fields [#184](https://github.com/AdRoll/baker/pull/184)
```

Comment:
There's a speedup also for some benchmarks with more fields, in any case i don't think we need to be so precise in the changelog
 ```suggestion
- Speedup `LogLine.ToText` and `Parse` methods [#184](https://github.com/AdRoll/baker/pull/184)
```